src/utils/simulation_results.jl

```diff
@@ -98,26 +127,47 @@ function load_simulation_results(
 end
 
 """
-    load_simulation_results(stage, SimulationResultsReference)
+    load_simulation_results(file_path, stage)
 
 This function goes through the reference table of file paths and
 aggregates the results over time into a struct of type OperationsProblemResults
 
 # Arguments
-- `SimulationResultsReference::SimulationResultsReference`: the container for the reference dictionary created in execute!
-- `stage_number::Int = 1`: The stage of the results getting parsed: 1 or 2
+- `file_path::String`: the file path to the dated folder with the raw results
+- `stage_number::String`: The stage of the results getting parsed
 
 # Example
 ```julia
-stage = 2
-step = ["step-1", "step-2", "step-3"] # has to match the date range
-variable = [:P_ThermalStandard, :P_RenewableDispatch]
-results = load_simulation_results(stage, step, variable, SimulationResultsReference)
+execute!(simulation)
+results = load_simulation_results("file_path", "stage_name")
 ```
 # Accepted Key Words
 - `write::Bool`: if true, the aggregated results get written back to the results file in the folder structure
+- `file_type::File Type = Feather`: default is feather file for writing the aggregated results back to the folder.
 """
+function load_simulation_results(path::String, stage_name::String, kwargs...)
+    sim_results = deserialize_sim_output(path)
+    load_simulation_results(sim_results, stage_name; kwargs...)
+end
+"""
+    load_simulation_results(SimulationResultsReference, stage)
+
+This function goes through the reference table of file paths and
+aggregates the results over time into a struct of type OperationsProblemResults
+
+# Arguments
+- `SimulationResultsReference::SimulationResultsReference`: the container for the reference dictionary created in execute!
+- `stage_number::String`: The stage of the results getting parsed
 
+# Example
+```julia
+sim_output = execute!(simulation)
+results = load_simulation_results(sim_output, "stage_name")
+```
+# Accepted Key Words
+- `write::Bool`: if true, the aggregated results get written back to the results file in the folder structure
+- `file_type::File Type = Feather`: default is feather file for writing the aggregated results back to the folder.
```

Comment:
Why would someone want to do this?

Reply:
@daniel-thom the simulation writes the results to a complicated folder structure, someone definitely might want to write a chunk of aggregated results back to the "results" folder.

Reply:
we have a distinction between results and raw_output. Results are the post-processed output according to the user's specification.